modules/apps/transfer/internal/convert/convert.go

```diff
@@ -1,54 +1 @@
 package convert
-
```

Comment:
moved because of import cycle. Ideally we only parse v1 denom when handling v1 packets (and there is only 1 string parsing function). The issue is that v1 packet data needs to parse v1 to do basic validation, so if you move v1 denom parsing here, you gain an import cycle